src/software/ai/ai.cpp

```diff
@@ -28,24 +27,6 @@ void Ai::overridePlayFromProto(const TbotsProto::Play& play_proto)
     overridePlay(std::move(createPlay(play_proto, strategy)));
 }
 
-void Ai::updateOverridePlay()
```

Comment:
does this mean there's a change to how AI is handled?

Reply:
No, just moved `updateAiConfig` from `Ai` to `ThreadedAi`. It's so that we don't get confused when writing tests and call `updateAiConfig` on `Ai` and also call `updateAiConfig` on `Strategy`